mne/preprocessing/maxwell.py

```diff
@@ -505,3 +537,107 @@ def _update_sss_info(raw, origin, int_order, ext_order, nsens):
         raw.info['proc_history'] = [proc_block]
 
     return raw
+
+
+def _read_fine_cal(info, fine_cal_fname):
+    """Read sensor locations and imbalances from fine calibration file."""
+
+    logger.info('Fine calibration file used in Maxwell Filter: ' +
+                fine_cal_fname)
+
+    # Read new sensor locations
+    with open(fine_cal_fname, 'r') as fid:
+        cal_chans = []
+        lines = fid.readlines()
+        lines = lines[::-1]
+        while len(lines) > 0:
```

Comment:
boo unnecessary `while` loop

`for line in lines` is clearer when you know the number of lines you're going to do. Save a couple lines by skipping the above lines and just doing:
```
for line in fid.readlines()[:-1]:
```